controller/appcontroller.go

```diff
@@ -188,8 +214,25 @@ func (ctrl *ApplicationController) setAppManagedResources(a *appv1.Application,
 func (ctrl *ApplicationController) getResourceTree(a *appv1.Application, managedResources []*appv1.ResourceDiff) (*appv1.ApplicationTree, error) {
 	nodes := make([]appv1.ResourceNode, 0)
 
+	proj, err := argo.GetAppProject(&a.Spec, applisters.NewAppProjectLister(ctrl.projInformer.GetIndexer()), ctrl.namespace)
+	if err != nil {
+		return nil, err
+	}
+	orphanedNodesMap := make(map[kube.ResourceKey]appv1.ResourceNode)
```

Comment:
could this have a better name? e.g. `keyToNode`

Reply:
I think `orphanedNodesMap` is easier to understand than `keyToNode`